yarn.lock

```diff
@@ -0,0 +1,568 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
While `yarn` is cool, I'd leave this out and keep only the `npm` version in. This makes it too opintionated